test/code_size/hello_wasm_worker_wasm.json

```diff
@@ -5,8 +5,8 @@
   "a.js.gz": 455,
   "a.ww.js": 115,
   "a.ww.js.gz": 127,
-  "a.wasm": 1894,
-  "a.wasm.gz": 1077,
-  "total": 3292,
-  "total_gz": 2043
+  "a.wasm": 1879,
+  "a.wasm.gz": 1066,
+  "total": 3277,
+  "total_gz": 2032
```

Comment:
This is great - I'm puzzled to understand how this saved code size though..?

Reply:
I'm also a little surprised.   

My initial guess was that perhaps `emscripten_stack_set_limits` was being included prior to this change and so this change reduces overall duplication... but I don't think thats true.  I could investigate further but but this seems like a win so I think we just take it.

Reply:
I think i double checked that.   I normally rebaseline all tests before landing changes like this, just to be sure I'm seeing the real deltas.